examples/step-29/step-29.cc

```diff
@@ -490,9 +492,11 @@ namespace Step29
       for (unsigned int face=0; face<GeometryInfo<dim>::faces_per_cell; ++face)
         if ( cell->face(face)->at_boundary() &&
              ((cell->face(face)->center() - transducer).norm_square() < 0.01) )
+          {
 
-          cell->face(face)->set_boundary_indicator (1);
-
+            cell->face(face)->set_boundary_indicator (1);
+            cell->face(face)->set_manifold_id (1);
```

Comment:
These two lines show an inconsistency in the naming convention we use in the library. I think this is my fault... 

Should we rename `set_manifold_id` to `set_manifold_indicator` and `manifold_id` to `manifold_indicator`?